model/dns_zone/dns_zone.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require_relative "../../model"
+
+class DnsZone < Sequel::Model
+  many_to_one :project
+  many_to_many :dns_servers
```

Comment:
I initially did that (I even add a similar recommendation to Furkan's patch), but then felt like "dns server" has a special meaning that we should keep conveying. I even used "name server" during the prototyping phase because that has even stronger meaning/connotation, but then changed it to dns server so that every model would be prefixed by dns. To me dns server provides a good balance between conveying the name server meaning and also being in harmony with the code around. Also server has close association with vm and one server backed by multiple vms sound weird, but one dns server backed by multiple vms doesn't sound weird that much.

However, this is all based on feeling :). If you prefer servers strongly, that might tip the balance for me as well.

Reply:
If you've already considered that, it's sufficient for me. I don't have strong feelings. I trust in yours.